docs/_config.yml

```diff
@@ -42,7 +42,7 @@ relative_links:
 # Handling Reading
 safe:         false
 include:      [".htaccess"]
-#exclude:      [_learn/ref]
+exclude:      [_learn/ref]
```

Comment:
Hey @lovettchris, thanks for the changes, btw was this one intended or leftover?

Reply:
woops, fixed, thanks.